<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="fedora" failures="1" tests="25" name="Default test" time="25.85" errors="0" timestamp="2022-08-13T01:46:05 EDT">
  <testcase classname="admin.AdminTest" name="EditBtnTest" time="1.398"/>
  <testcase classname="admin.AdminTest" name="validateHeadersTest" time="0.148"/>
  <testcase classname="admin.AdminTest" name="validatePageTitleTest" time="0.296"/>
  <testcase classname="admin.AdminTest" name="validatePriceTest" time="0.334"/>
  <testcase classname="shop.ShopTest" name="buttonNotLoginTest" time="0.826"/>
  <testcase classname="shop.ShopTest" name="buttonTest" time="0.446"/>
  <testcase classname="shop.ShopTest" name="categoryTest" time="0.974"/>
  <testcase classname="shop.ShopTest" name="prodBuyBtnNotLoginTest" time="0.444"/>
  <testcase classname="shop.ShopTest" name="productListTest" time="3.664"/>
  <testcase classname="shop.ShopTest" name="searchTest" time="0.363"/>
  <testcase classname="shop.ShopTest" name="validatePageTitleTest" time="0.169"/>
  <testcase classname="login.LoginTest" name="badLoginTest" time="1.513"/>
  <testcase classname="login.LoginTest" name="testEmptyPassword" time="0.319"/>
  <testcase classname="login.LoginTest" name="testEmptyUsername" time="0.246"/>
  <testcase classname="login.LoginTest" name="validatePageTitleTest" time="0.423"/>
  <testcase classname="login.RegistrationTest" name="ExistingUserTest" time="3.918"/>
  <testcase classname="login.RegistrationTest" name="InvalidPasswodTest" time="0.758"/>
  <testcase classname="login.RegistrationTest" name="ValidRegistrationTest" time="5.848"/>
  <testcase classname="login.RegistrationTest" name="disableBtnTest" time="0.563"/>
  <testcase classname="login.RegistrationTest" name="invalidEmailTest" time="0.68"/>
  <testcase classname="login.RegistrationTest" name="validatePageTitleTest" time="0.213"/>
  <testcase classname="admin.AdminEditProductTest" name="testProductId" time="0.833">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;ID&amp;gt; but was:&amp;lt;ID 1&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<ID> but was:<ID 1>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:125)
at admin.AdminEditProductTest.testProductId(AdminEditProductTest.java:61)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testProductId -->
  <testcase classname="admin.AdminEditProductTest" name="validateFormTest" time="0.369"/>
  <testcase classname="admin.AdminEditProductTest" name="validatePageTitleTest" time="0.207"/>
  <testcase classname="admin.AdminEditProductTest" name="validateSuccessMessageTest" time="0.463"/>
</testsuite> <!-- Default test -->
